src/formatter/Formatter.hx

```diff
@@ -108,3 +175,9 @@ class Formatter {
 		return null;
 	}
 }
+
+enum FormatterInput {
+	FileInput(fileName:String, config:Config, ?lineSeparator:String, ?entryPoint:TokenTreeEntryPoint);
+	Code(code:String, fileName:String, config:Config, ?lineSeparator:String, ?entryPoint:TokenTreeEntryPoint);
+	Tokens(tokenList:Array<Token>, tokenTree:TokenTree, code:String, fileName:String, config:Config, ?lineSeparator:String, ?entryPoint:TokenTreeEntryPoint);
```

Comment:
I don't think this fits the current usage of haxe-language-server? It supplies the bytes directly, rather than passing the code as a string.